# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: ef817155231f
# Full Hash: ef817155231f6fed522ce793cbc82b7be019de56
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Constify conduit getters. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D122244
# ==============================================================================

diff -r 310fdf39b051 -r ef817155231f dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Sun Apr 18 21:25:11 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Tue Aug 10 14:58:34 2021 +0200
@@ -114,7 +114,7 @@
   return RecreateSendStreamIfExists();
 }
 
-std::vector<uint32_t> WebrtcAudioConduit::GetLocalSSRCs() {
+std::vector<uint32_t> WebrtcAudioConduit::GetLocalSSRCs() const {
   MOZ_ASSERT(mCallThread->IsOnCurrentThread());
   return std::vector<uint32_t>(1, mRecvStreamConfig.rtp.local_ssrc);
 }
@@ -131,7 +131,7 @@
   return RecreateRecvStreamIfExists();
 }
 
-bool WebrtcAudioConduit::GetRemoteSSRC(uint32_t* ssrc) {
+bool WebrtcAudioConduit::GetRemoteSSRC(uint32_t* ssrc) const {
   MOZ_ASSERT(mCallThread->IsOnCurrentThread());
   if (!mRecvStream) {
     return false;